bson/__init__.py

```diff
@@ -269,10 +267,12 @@ def _get_string(
     length = _UNPACK_INT_FROM(data, position)[0]
     position += 4
     if length < 1 or obj_end - position < length:
-        raise InvalidBSON("invalid string length")
+        msg = "invalid string length"
+        raise InvalidBSON(msg)
```

Comment:
Ew I don't like what ruff is doing here at all. Looking for a code to disable.

Reply:
That rule comes from https://pypi.org/project/flake8-errmsg/

Reply:
https://beta.ruff.rs/docs/rules/#flake8-errmsg-em

Reply:
Caused by https://beta.ruff.rs/docs/rules/#flake8-errmsg-em

Disabling now.